### DISCUSSION TITLE
How to contact Github Support if 2FA is not working

### DISCUSSION THREAD

**Discussion by aalborz** [community/community#101450] [Other Feature Feedback, Questions, & Ideas]
I can't login to our corporate Github account because the person who opened up the account has left and I don't have access to 2FA used.
There's no phone number to contact github and in order to contact support I have to login first, which is catch 22!!!!
Anybody knows how to contact their support without having to go through the hoops and not getting anywhere???

[a comment by an automated account is not shown]

**Comment by ghostinhershell** [comment]
Hi @aalborz, thanks so much for reaching out in our community. 

We do have a support article that documents next steps if you lose access to your two-factor authentication credentials: [Recovering your account if you lose your 2FA credentials](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials). I think this section, specifically, should be helpful: [Requesting help with two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#requesting-help-with-two-factor-authentication).

If that doesn't help, you can submit a support ticket via our [Support page](https://support.github.com/contact).
